pymc/tests/test_idata_conversion.py

```diff
@@ -602,6 +602,15 @@ def test_issue_5043_autoconvert_coord_values(self):
             )
             assert isinstance(converter.coords["city"], pd.MultiIndex)
 
+    @pytest.mark.xfail(
```

Comment:
The test should use a `pytest.raises` context and not be marked as xfail. See https://docs.pytest.org/en/6.2.x/assert.html#assertions-about-expected-exceptions